Improve modular transformers documentation (#35322)
huggingface/transformers: docs/source/en/modular_transformers.md

### Details 

> To generate the single-files from the modular file, run `python utils/modular_model_converter.py --files-to-parse src/transformers/models/<your_model>/modular_<your_model>.py`.

Reviewer: Nicer to put the command in a copyable code cell :)

```suggestion
To generate a single file from the modular file, run the following command.

```bash
python utils/modular_model_converter.py --files-to-parse src/transformers/models/<your_model>/modular_<your_model>.py
```
```

Author: Yes, thanks, this is better 👍

### Enforcement

> You can run `python utils/check_modular_conversion.py --files src/transformers/models/<your_model>/modular_<your_model>.py` to make sure the generated content matches what is present in `modular_<your_model>.py`

Author: I'm not sure if this is accurate since it fails in my case (#35320).
I'm assuming this is the check the TODO is referring to, please double check.

> You can run `python utils/check_modular_conversion.py --files src/transformers/models/<your_model>/modular_<your_model>.py` to make sure the generated content matches what is present in `modular_<your_model>.py`

Reviewer: cc @ArthurZucker 

```suggestion
Run the command below to ensure the generated content matches `modular_<your_model>.py`

```bash
python utils/check_modular_conversion.py --files src/transformers/models/<your_model>/modular_<your_model>.py
```
```
